supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -258,21 +258,37 @@ object TesTask {
     )
   }
 
-  def makeTask(tesTask: TesTask): Task = {
+  def makeTask(tesTask: TesTask, transformBlobToLocalPath: Boolean = false): Task = {
+    val inputs = if (transformBlobToLocalPath) transformInputs(tesTask.inputs) else tesTask.inputs
+    val outputs = if (transformBlobToLocalPath) transformOutputs(tesTask.outputs) else tesTask.outputs
     Task(
       id = None,
       state = None,
       name = Option(tesTask.name),
       description = Option(tesTask.description),
-      inputs = Option(tesTask.inputs),
-      outputs = Option(tesTask.outputs),
+      inputs = Option(inputs),
+      outputs = Option(outputs),
       resources = Option(tesTask.resources),
       executors = tesTask.executors,
       volumes = None,
       tags = Option(tesTask.jobDescriptor.workflowDescriptor.customLabels.asMap),
       logs = None
     )
   }
+
+  def transformInputs(inputs: Seq[Input]): Seq[Input] = inputs.map(i =>
+    i.copy(url=i.url.map(s => transformBlobString(s)))
+  )
+
+  def transformOutputs(outputs: Seq[Output]): Seq[Output] = outputs.map(i =>
+    i.copy(url=i.url.map(s => transformBlobString(s)))
```

Comment:
This reduces to
```
i.copy(url=i.url.map(transformBlobString(_)))
```
which reduces to
```suggestion
    i.copy(url=i.url.map(transformBlobString))
```

Reply:
Thank you, I think this was leftover from a moment when I was passing something else into that function.